locallib.php

```diff
@@ -1306,6 +1311,7 @@ function moodleoverflow_print_post($post, $discussion, $moodleoverflow, $cm, $co
 
     // Prepare the post.
     $mustachedata->postcontent = format_text($post->message, $post->messageformat, $options, $course->id);
+    $mustachedata->commentcontent = strip_tags($post->message, "<style>");
```

Comment:
If I leave the html tags in, the text editor inserts a break after the comment making it bigger in size. I am not sure how else I am supposed to prevent that from happening. Do you have a better idea?

Reply:
I think a single br at the end of a tag does not change anything (br forces linebreaks *between* textparts) . The html edit sometimes (I think when fully loaded before beginning to type) wraps the text inside \<p\>, which has a small margin at the bottom, this only appears to change the size of the comment with two or more text lines.

I think we want the use to be able to use the features of the html editor, and let them format and style their texts, so we can't strip all tags. Theoretically, there would be a option like removing the bottom-margin of the last \<p\> tag inside the .comment-text (with a CSS rule), but that seems a bit hacky. I personally think that small amount of margin is not really a problem, because comments are now already more subtle than answers.

Reply:
okay, i removed it. to me it just looks better without the margin, but that might just be me :)

Reply:
I mean you're definitely right, it looks a bit better, but I think not enough that it would be worth the effort trying to achieve removing that margin consistently (in a non-hacky way), while the text is being formatted ^^